spec/php-spec.coffee

```diff
@@ -820,11 +820,12 @@ describe 'PHP grammar', ->
     it 'tokenizes function names with characters other than letters or numbers', ->
       # The space between foo and bar is a nbsp (char 160/hex 0xA0), not an actual space (char 32/hex 0x20)
       # 0xA0 is between 0x7F and 0xFF, making it a valid PHP identifier
-      tokens = grammar.tokenizeLines "<?php\nfunction foo bar() {}"
+      functionName = "foo#{String.fromCharCode 160}bar"
```

Comment:
:zap:
You can probably condense the above comment to just say that 160 is allowed because its hex representation is between 0x7f-0xff.